src/GslCore/LibraryPartProvider.fs

```diff
@@ -0,0 +1,30 @@
+﻿/// Support for reusable part retrieval based on a FASTA file.
+module LibraryPartProvider
+open System.IO
+open commonTypes
+open PluginTypes
+open Amyris.Dna
+open Amyris.ErrorHandling
+
+type LibraryPartProvider() =
```

Comment:
Like this! - have been wanting to just get rid of this legacy concept but this is probably a more responsible approach.  Not sure if that feature is in use at Amyris.  I don't think we are using it at Demetrix.

Reply:
I don't think it's in use at Amyris beyond regression tests.  I think as a plugin is provides a relatively easy feature for anyone who wants to use the compiler to provide a library of components (just need a fasta file), and serves as a simple template for others to implement part providers.